src/exporters/admin_boundaries.py

```diff
@@ -0,0 +1,132 @@
+from pathlib import Path
+import os
+from .base import BaseExporter
+
+
+class OCHAExporter(BaseExporter):
+    """Export Administrative boundaries for country from the
+    Organisation for the Coordination of Humanitarian Affairs.
+
+    https://data.humdata.org/
+    """
+    country: str
+
+    def __init__(self, data_folder: Path = Path('data')) -> None:
+        super().__init__(data_folder)
+        # write the download to landcover
+        self.admin_bounds_folder = self.raw_folder / 'boundaries' / self.country
+        if not self.admin_bounds_folder.exists():
+            self.admin_bounds_folder.mkdir(parents=True, exist_ok=True)
+
+    def wget_file(self, url_path: str) -> None:
+        output_file = self.admin_bounds_folder / url_path.split('/')[-1]
+        if output_file.exists():
+            print(f'{output_file} already exists! Skipping')
+            return None
+
+        os.system(f'wget {url_path} -P {self.admin_bounds_folder.as_posix()}')
+
+    def unzip(self, fname: Path) -> None:
+        print(f'Unzipping {fname.name}')
+
+        os.system(f'unzip {fname.as_posix()} -d {self.admin_bounds_folder.resolve().as_posix()}')
+        print(f'{fname.name} unzipped!')
+
+    def rename_dirs(self, fname, new_fname) -> None:
+        """TODO: need to rename the unzipped file NOT the zipped file"""
+        new_fname = new_fname + '.zip' if new_fname[-4:] != '.zip' else new_fname
+        new_filepath = fname.absolute().parents[0] / new_fname
+        os.system(f'mv {fname.absolute().as_posix()} {new_filepath.as_posix()}')
+        print(f'Renamed file to: {new_fname.as_posix()}')
+
+
+class KenyaAdminExporter(OCHAExporter):
+    country = 'kenya'
+
+    def export(self) -> None:
+        """Export functionality for the Kenya Admin Boundaries as .shp files
+        """
+        urls = {
```

Comment:
nit: Should these URLs be class attributes?

Reply:
so they can be accessed as `self.urls`?

Reply:
so they can be accessed as `self.urls`?

if you think so! yeah for sure

Reply:
okay fixed that!